docu/files/SkeletonCreatorBuilderCLI.html.md

```diff
@@ -1,45 +1,65 @@
+<img style="width: 30%;" max-width: 326px src="img/skeletor_ui_drag.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
+
+5. After you create the view, click the preview button to the right of the delete button.  The browser immediately stores your view in the local cache. That means you will not loose the view until you delete it by explicitly clicking on the delete button.
 
-  <img src="img/skeletor_ui_menu.png" width="35%" class="img-click-modal" alt="The Skeleton Creator UI"/><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_ui_menu.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br>
 
 6. Export the code that the Skeleton Creator generates by clicking on the show button to the right of the preview button.
 
 Here is an example of a UI with its corresponding code preview:<br>
 
-  <img src="img/skeletor_ui_completed.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
-  <img src="img/skeletor_code_sample.png" class="img-click-modal" alt="Code Preview"/><br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_ui_completed.png" class="img-click-modal" alt="The Skeleton Creator UI"/><br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_code_sample.png" class="img-click-modal" alt="Code Preview"/><br><br>
 
-  If you are using Chrome or Firefox, a download button is available in preview mode allowing you to download the code to a file.
+If you are using Chrome or Firefox, a download button is available in preview mode allowing you to download the code to a file.
 
-  <img src="img/skeletor_menu_download_button.png"  width="35%" class="img-click-modal" alt="The Skeleton Creator download button"/><br><br>
-  Internext Explorer and Safari do not support the download button. Instead, you can copy the HTML code directly from the broswer. <br><br>
+<img style="width: 30%;" max-width: 326px src="img/skeletor_menu_download_button.png"  width="35%" class="img-click-modal" alt="The Skeleton Creator download button"/><br><br>
+Internext Explorer and Safari do not support the download button. Instead, you can copy the HTML code directly from the broswer. <br><br>
 
 
-<h3>Component settings</h3>
+**Component settings**
 
 You can adjust the settings for each component in the Skeleton Creator. This allows you to assign custom text and and labeling to the component. For example, the <strong>Page Header</strong> component features a <strong>Back</strong> button, a long page title, as well as a <strong>Save</strong> and <strong>Cancel</strong> button by default. You can set the default text of the title and rename each of the buttons.
 
 <br>
-  <img src="img/component_settings.png" class="img-click-modal" alt="Code Preview"/>
+  <img style="width: 30%;" max-width: 326px src="img/component_settings.png" class="img-click-modal" alt="Code Preview"/><br><br>
+  
+
+<h3>Dev Mode to preview your module</h3>
```

Comment:
Start headings with an active verb when possible: 

<h3>Preview your module in Dev Mode</h3>